testsuite/test_api.py

```diff
@@ -138,7 +138,7 @@ def test_styleguide(self):
         self.reset()
 
     def test_styleguide_options(self):
-        # Instanciate a simple checker
+        # Instanstiate a simple checker
```

Comment:
Once you update this and revert the change to behaviour this will be good to go.